acceptance-tests/pubsub_test.go

```diff
@@ -0,0 +1,45 @@
+package acceptance_test
+
+import (
+	"csbbrokerpakgcp/acceptance-tests/helpers/apps"
+	"csbbrokerpakgcp/acceptance-tests/helpers/matchers"
+	"csbbrokerpakgcp/acceptance-tests/helpers/random"
+	"csbbrokerpakgcp/acceptance-tests/helpers/services"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("PubSub", Label("pubsub"), func() {
+	It("can be accessed by an app", func() {
+		By("creating a service instance")
+		serviceInstance := services.CreateInstance(
+			"csb-google-pubsub",
+			"default",
+			services.WithParameters(map[string]any{"subscription_name": random.Name()}))
+		defer serviceInstance.Delete()
+
+		By("pushing the unstarted app twice")
+		publisherApp := apps.Push(apps.WithApp(apps.PubSubApp))
+		subscriberApp := apps.Push(apps.WithApp(apps.PubSubApp))
+		defer apps.Delete(publisherApp, subscriberApp)
+
+		By("binding the apps to the storage service instance")
+		binding := serviceInstance.BindWithParams(publisherApp, `{"role":"pubsub.editor"}`)
```

Comment:
This is the default role. I'd be inclined to leave it out, or do a more interesting test with non-default roles.

Reply:
Oh yeah I tried doing one as "publisher" role and one as "subscriber". It did highlight issues that I will look into tomorrow.